src/Articles/containers/CreateArticle.jsx

```diff
@@ -0,0 +1,321 @@
+import {
+  Form, Row, Col, Button, ProgressBar,
+} from 'react-bootstrap';
+import React from 'react';
+import '../CSS/CreateArticle.scss';
+import { connect } from 'react-redux';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faCamera } from '@fortawesome/free-solid-svg-icons';
+import Loader from 'react-loader';
+import PropTypes from 'prop-types';
+import { storage } from '../../commons/Util/FirebaseConfig';
+import postAnArticle from '../redux/actions/CreateArticleAction';
+import ErrorLabel from '../../commons/components/ErrorLabel';
+import SuccessLabel from '../../commons/components/SuccessLabel';
+import { TITLE_REGEX, BODY_REGEX, TAGS_REGEX } from '../constants';
+
+export class CreateArticle extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      isFieldInvalid: {
+        title: null,
+        body: null,
+        tags: null,
+      },
+      articleTitle: '',
+      articleBody: '',
+      articleTags: '',
+      uploadProgress: 0,
+      displayProgress: 'none',
+      isFormValid: false,
+      articlePhoto: {
+        name: 'No Photo selected',
+        data: null,
+      },
+    };
+
+    this.handleCreateArticle = this.handleCreateArticle.bind(this);
```

Comment:
Do you still have to bind even when using arrow functions?

Reply:
@AGMETEOR thanks for this. I had no idea about it but I have fixed it.